backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -176,6 +185,10 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
                     tasks = tasks.Concat(_service.Task.GetForWorkflow("ERP"));
                     tasks = tasks.Concat(_service.Task.GetForWorkflow("SPL"));
                     project.AddTask(tasks.DistinctBy(t => t.Id).ToArray());
+                    project.Tasks.ForEach(t =>
+                    {
+                        t.IsCompleted = !t.Task.IsOptional;
```

Comment:
Default project tasks that are required to completed.  Otherwise they are permanently stuck.